insights/parsers/ansible_tower_config_custom.py

```diff
@@ -0,0 +1,44 @@
+"""
+AnsibleTowerConfigCustom - file ``/etc/tower/conf.d/custom.py``
+===============================================================
+The AnsibleTowerConfigCustom class parses the file ``/etc/tower/conf.d/custom.py``.
+"""
+from insights import parser, get_active_lines, Parser, LegacyItemAccess
+from insights.core.filters import add_filter
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+add_filter(Specs.ansible_tower_config_custom, ["AWX_CLEANUP_PATHS", "="])
+
+
+@parser(Specs.ansible_tower_config_custom)
+class AnsibleTowerConfigCustom(Parser, LegacyItemAccess):
```

Comment:
Since this parser will be storing key/value pairs it really should be derived from `dict` instead of `LegacyItemAccess`.  Here is [an example](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/partitions.py) a parser implementing `dict`.

Reply:
@bfahr - Good to me.